lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -1,25 +1,52 @@
 # frozen_string_literal: true
 module Common
   module Exceptions
-    # BackendServiceException - This will eventually receive a Common::Client::Errors:BackendServiceError
-    # It should not be invoked directly by clients at any time. The Client:Error should be
-    # rescued and this should be raised instead.
+    # BackendServiceException - This will return a generic error, to customize
+    # you must define the minor code in the locales file and call this class from
+    # raise_error middleware.
     class BackendServiceException < BaseError
-      def initialize(error_klass = nil, options = {})
-        @error_klass = error_klass || 'client_error'
-        @detail = options[:detail]
-        @source = options[:source]
-        @meta = options[:meta] unless Rails.env.production?
+      attr_reader :response_values
+
+      def initialize(key = nil, response_values = {})
+        if response_values[:status]&.between?(400, 999)
+          @response_values = response_values
+        else
+          raise NotImplementedError, 'only invoke from raise_error middleware'
+        end
+        @key = key || 'backend_service_exception'
+      end
+
+      def message
+        "BackendServiceException: #{response_values}"
       end
 
       def errors
-        Array(SerializableError.new(i18n_data.merge(detail: @detail, source: @source, meta: @meta)))
+        Array(SerializableError.new(i18n_data.merge(status: _status, detail: _detail, source: _source)))
       end
 
       private
 
+      # The http status code. This is very important and required for rendering
+      # unless you've specified that you want the status code to be something other
+      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
+      # was provided by the backend service, because the backend service response
+      # might not always be rele
+      def _status
```

Comment:
Yea, I have seen the underscore pattern used before when there is a semantic difference in meaning to what the class is designed to do. I could just remove the underscore and keep them private. My syntax highlighting in atom also makes it think as though "source" is a reserved word. Which is mildly annoying.